testing_and_setup/compass/utility_scripts/LIGHTparticles/make_particle_file.py

```diff
@@ -348,26 +349,74 @@ def write(self, f_name, f_decomp):  #{{{
 #}}}
 
 
-def get_cell_coords(f_init): #{{{
-    return f_init.variables['xCell'][:], \
-           f_init.variables['yCell'][:], \
-           f_init.variables['zCell'][:] #}}}
+def get_particle_coords(f_init, loc=None): #{{{
+    #
+    if loc is None:
+        loc = 'center'
+
+    xCell = f_init.variables['xCell'][:]
+    yCell = f_init.variables['yCell'][:]
+    zCell = f_init.variables['zCell'][:]
+
+    if loc == 'center':
+        cells = (xCell, yCell, zCell)
+        cpts = np.arange(len(xCell))
+    elif loc == 'offvertex':
+        xVertex = f_init.variables['xVertex'][:]
+        yVertex = f_init.variables['yVertex'][:]
+        zVertex = f_init.variables['zVertex'][:]
+
+        cellsOnVertex = f_init.variables['cellsOnVertex'][:,:]
+        areaCell = f_init.variables['areaCell'][:]
```

Comment:
@pwolfram, this can also be deleted. It was used in dsCell and is used nowhere else in the script.

Reply:
Done, thanks!